vpr/src/place/initial_placement.cpp

```diff
@@ -417,6 +419,9 @@ void initial_placement(enum e_pad_loc_type pad_loc_type, const char* constraints
     vtr::vector<ClusterBlockId, t_block_score> block_scores = assign_block_scores();
     std::vector<ClusterBlockId> sorted_blocks = sort_blocks(block_scores);
 
+    //Perform constraints_propagation
```

Comment:
Not a great comment as it is pretty close to just the function name.  Explain what constraints_propagation is briefly (go through floorplan and .locked constraints to ensure we have the tightest constraint on each block when there are interacting constraints).

Reply:
propagate_placement_cons or propagate_place_constraints() are better names I think; they read as an action.